ui/src/styles.scss

```diff
@@ -53,13 +53,18 @@ body {
 }
 
 .warning {
-  color: #f3b300;
+  color: #ff9800;
 }
 
 .success {
   color: green;
 }
 
+.suggestion {
+  color: #f6be00;
+  transform: rotate(180deg);
+}
+
```

Comment:
these css changes do not seem to be part of this PR same for restore index PR

Reply:
Updated